db/migrate/20170909151007_add_title_to_versions.rb

```diff
@@ -0,0 +1,5 @@
+class AddTitleToVersions < ActiveRecord::Migration[5.1]
+  def change
+    add_column :versions, :title, :string
```

Comment:
I think we should also make sure we fill in the `title` attribute on all existing versions with the existing title from their page. You could do that here or make a second migration for it.

Reply:
Makes sense to me. I just went ahead and added that to this existing migration.